# An surface wind map

Header header
MeanWind wind
Raster raster